index.php

```diff
@@ -55,7 +55,10 @@
 		$series = array_rand($titles);
 		$title = array_rand($titles[$series]);
 		$video = $titles[$series][$title];
-		$filename = $video['file'];
+        $filename = $video['file'];
+        if(!array_key_exists("mime", $video)){
+            $video["mime"]= ['video/webm;codecs="vp9,opus"'];
```

Comment:
Just `'video/webm'` would be better. We don't know what video and audio formats were used; only that they were in webm files.